src/App.js

```diff
@@ -1,9 +1,9 @@
-import React, { useState, useEffect, useMemo, useCallback } from 'react';
+import React, { useState, useEffect } from 'react';
```

Comment:
I'd say it's better not to add the hooks yet as that should also be part of the workshop: adding them in the first place.